README.md

```diff
@@ -20,7 +20,7 @@ repositories {
     jcenter()
 }
 dependencies {
-    implementation 'com.inmobi.monetization:inmobi-ads:9.0.9'
+    implementation 'com.inmobi.monetization:inmobi-ads:9.1.0'
```

Comment:
@rishi-jha-inm
 **## New in this version**
Also needs updation with the latest changes